zubhub_frontend/zubhub/package.json

```diff
@@ -2,56 +2,62 @@
   "name": "zubhub",
   "version": "0.1.0",
   "private": true,
+  "peerDependencies": {
```

Comment:
https://mui.com/material-ui/getting-started/installation/#peer-dependencies. I hope that settles this, otherwise, I will have to make it a direct dependency as you've said

Reply:
I've seen the link you provided before. By that they meant that if for some reason you don't use react in your project but still intend to use mui, you have to add react and react-dom as peerDependencies. in our case we already have (and still have them) in dependencies. This introduces the opportunity to have a potential bug where `peerDependency` react is different from `dependency` react.
If we must have this then we have to remove `react` and `react-dom` from `dependencies` so we know the only source of truth is `peerDependencies`

Reply:
okay @NdibeRaymond, thanks